tracing-subscriber/tests/reload.rs

```diff
@@ -1,4 +1,4 @@
-#![cfg(feature = "reload")]
+#![cfg(feature = "registry")]
```

Comment:
I think its impossible to run this test without this, on master we probably need this change as well

Reply:
it should probably be 
```suggestion
#![cfg(all(feature = "registry", feature = "reload"))]
```
because it does also require the "reload" feature, right?

Reply:
...actually, looks like only one of the tests requires the "registry" feature, so i would prefer to only put the feature flag there.
```suggestion
#![cfg(feature = "reload")]
```